packages/manager/modules/pci/src/projects/project/failover-ips/onboarding/order/order.routing.js

```diff
@@ -19,22 +15,50 @@ export default /* @ngInject */ ($stateProvider, coreConfigProvider) => {
     resolve: {
       breadcrumb: /* @ngInject */ () => null,
       /* @ngInject */
-      products: (OvhApiOrderCatalogFormatted) =>
+      products: /* @ngInject */ (
+        availableRegions,
+        OvhApiOrderCatalogFormatted,
+      ) =>
         OvhApiOrderCatalogFormatted.v6()
           .get({
             catalogName: 'ip',
             ovhSubsidiary: coreConfigProvider.getUser().ovhSubsidiary,
           })
           .$promise.then(({ plans }) =>
-            filter(
-              plans,
+            plans.filter(
               (offer) =>
                 /failover/.test(offer.planCode) &&
-                offer.invoiceName.includes(
-                  get(REGIONS, coreConfigProvider.getRegion()),
+                availableRegions.some((region) =>
+                  offer.invoiceName.includes(region),
                 ),
             ),
           ),
+      continentCodeEnum: /* @ngInject */ ($http) =>
+        $http
+          .get('/cloud.json')
+          .then(
+            ({ data: schema }) =>
+              schema.models['cloud.RegionContinentEnum'].enum,
+          ),
+      availableRegions: /* @ngInject */ (
+        continentCodeEnum,
+        iceberg,
+        projectId,
+      ) =>
+        iceberg(`/cloud/project/${projectId}/region`)
+          .query()
+          .expand('CachedObjectList-Pages')
+          .execute()
+          .$promise.then(({ data: regions }) => {
+            const continentCodes = regions.flatMap(
+              ({ continentCode }) => continentCode,
+            );
+
+            return continentCodeEnum
+              .filter((continentCode) => continentCodes.includes(continentCode))
```

Comment:
`continentCodeEnum` is useless, we could retrieve a Set of continentCodes from region and use it.

Reply:
Good catch, fixed